projects/file-parsing/types-converting/custom-binary-struct.go

```diff
@@ -0,0 +1,31 @@
+package filesConverting
+
+import (
+	"encoding/binary"
+)
+
+func CustomBinaryToStruct(textByte []byte) []Score {
+	result := []Score{}
+	s := Score{}
+
+	i := 2
+	for i < len(textByte) {
+		j := i + 4
+		if textByte[0] == 255 && textByte[1] == 254 { // little endian byte order
```

Comment:
Can you think of a way to avoid doing these comparisons so often, and also avoid the need for this comment?